website/src/pages/ManufacturedComponentsPage.tsx

```diff
@@ -59,95 +59,33 @@ const components: ManufacturedComponent[] = [
+  { header: 'Quantity', key: 'quantity' },
+  { header: 'Unit Price (JPY)', key: 'unitPrice' },
+  { header: 'Total Price (JPY)', key: 'totalPrice' },
+  { header: 'Manufacturing Method', key: 'method' },
+  { header: 'Material', key: 'material' },
+  { header: 'Manufacturer', key: 'manufacturer' }
+];
 
+export default function ManufacturedComponentsPage(): ReactNode {
+  const totalCost = formatTotalCost(components);
+  
   return (
-    <div style={{ overflowX: 'auto' }}>
-      <p>
-        <ins>
-          {`The estimated total cost of manufactured components is ${calculateTotalCost()}`}
-        </ins>
-      </p>
+    <div>
+      <h1>Manufactured Components</h1>
       <p>
-        Here is the list of the components that need to be manufactured:
+        <ins>The estimated total cost of manufactured components is {totalCost}</ins>
       </p>
-      <table style={tableStyles.table}>
-        <thead>
-          <tr style={tableStyles.headerRow}>
-            <th style={tableStyles.headerCell}>Name</th>
-            <th style={tableStyles.headerCell}>Photo</th>
-            <th style={tableStyles.headerCell}>Model Number</th>
-            <th style={tableStyles.headerCell}>Quantity</th>
-            <th style={tableStyles.headerCell}>Unit Price (JPY)</th>
-            <th style={tableStyles.headerCell}>Total Price (JPY)</th>
-            <th style={tableStyles.headerCell}>Manufacturing Method</th>
-            <th style={tableStyles.headerCell}>Material</th>
-            <th style={tableStyles.headerCell}>Manufacturer</th>
-          </tr>
-        </thead>
-        <tbody>
-          {components.map((component) => (
-            <tr key={component.name} style={tableStyles.bodyRow}>
-              <td style={tableStyles.bodyCell}><strong>{component.name}</strong></td>
-              <td style={tableStyles.bodyCell}>
-                <ComponentPhoto
-                  src={useBaseUrl(`/img/bom/manufactured_components/${component.image}`)}
-                  alt={component.name}
-                />
-              </td>
-              <td style={tableStyles.bodyCell}>{component.model}</td>
-              <td style={tableStyles.bodyCell}>{component.quantity}</td>
-              <td style={tableStyles.bodyCell}>{formatPrice(component.unitPrice)}</td>
-              <td style={tableStyles.bodyCell}>
-                <strong>{formatPrice(component.unitPrice * component.quantity)}</strong>
-              </td>
-              <td style={tableStyles.bodyCell}>{component.method}</td>
-              <td style={tableStyles.bodyCell}>{component.material}</td>
-              <td style={tableStyles.bodyCell}>{component.manufacturer}</td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
+      <p>Here is the list of the components that need to be manufactured:</p>
+      <ComponentTable
+        components={components}
+        columns={columns}
+        imageBasePath="/img/bom/manufactured_components"
```

Comment:
Clould you use `-` for word separator in file name? 

```suggestion
        imageBasePath="/img/bom/manufactured-components"
```